src/ts/common/ui/modal.ts

```diff
@@ -24,9 +24,10 @@ interface ModalInput extends ModalElement {
 interface ModalOptions {
 	text: string;
 	subText?: string[];
-	elements: (ModalButton | ModalInput)[];
+	elements?: (ModalButton | ModalInput)[];
```

Comment:
idk if we want to do this, i just didn't need it for my use case

Reply:
Actually, I think a button to open the options would be a good idea! You can use `invokeWorker(OpenOptions)` or something like that to open them.

Reply:
Gonna use this in the docs
<img width="1335" alt="image" src="https://user-images.githubusercontent.com/35436247/211142824-1299cd32-bff8-43c3-adde-720f0ba6f251.png">

Reply:
aaa! `tag index` to `Tag Index` here too

Reply:
ok gonna use _this_ in the documentation

<img width="1335" alt="image" src="https://user-images.githubusercontent.com/35436247/211142897-4b404dfc-327d-4738-9e73-dd27427a073d.png">